File: src/main/java/io/github/dqualizer/dqlang/draft/dam/ServerInfo.java
package io.github.dqualizer.dqlang.draft.dam;

import lombok.Data;
import org.bson.types.ObjectId;

@Data
public class ServerInfo {

    private String id;
    private String host;
    private String environment;

    public ServerInfo() {
        this.id = new ObjectId().toHexString();
    }
}

File: src/main/kotlin/dqualizer/dqlang/archive/openapiadapter/dqlang/ServerInfo.kt
package dqualizer.dqlang.archive.openapiadapter.dqlang

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class ServerInfo {
    private val host: String? = null
    private val environment: String? = null
}
